### ISSUE TITLE
Local ruff check does not match up with remote ruff check

### ISSUE THREAD

**Issue by norswap** [0xFableOrg/simple-op-stack-rollup#16]
When running the check on #12 (before I fixed it) ([transcript](https://github.com/0xFableOrg/simple-op-stack-rollup/actions/runs/5905456834/job/16019576850?pr=12)):

```
Successfully installed ruff-0.0.285
libroll.py:55:45: E721 Do not compare types, use `isinstance()`
Found 1 error.
make: *** [Makefile:2: linter] Error 1
```

Locally I have ruff 0.0.284, which the `install.sh` script is happy to accept. We should make sure that if the users reruns `source scripts/install.sh` then the results of `ruff` matches up to the server (by constraining version in both of these places?).

**Comment by GrapeBaBa** [comment]
it seems ruff releasing very often. If we always use latest version, we will get lint issues even no code/minor code change. Probably we need focus a version right now? @norswap @ayazabbas @eerkaijun

**Comment by norswap** [comment]
Yes, let's just settle on the latest version right now, and we'll update it periodically :)
